astroid/tests/unittest_brain.py

```diff
@@ -548,6 +548,74 @@ def test_sys_streams(self):
             self.assertIsInstance(raw, astroid.Instance)
             self.assertEqual(raw.name, 'FileIO')
 
+class TypingBrain(unittest.TestCase):
```

Comment:
Technically typing is available in python2 also but in pypi.  Though for tests maybe I should limit it to 3.6